server/conn.go

```diff
@@ -690,6 +690,7 @@ func (cc *clientConn) Run(ctx context.Context) {
 				zap.String("command", mysql.Command2Str[data[0]]),
 				zap.String("status", cc.SessionStatusToString()),
 				zap.String("sql", queryStrForLog(string(data[1:]))),
+				zap.String("txn_mode", cc.ctx.GetSessionVars().GetReadableTxnMode()),
```

Comment:
Help me check if there is risk of nil pointer dereference

Reply:
any update?

Reply:
I think it could not be nil, PTAL @coocood .

Reply:
if handleChangeUser failed, cc.ctx may be nil